doc/classes/Viewport.xml

```diff
@@ -270,6 +266,12 @@
 		</member>
 		<member name="snap_2d_vertices_to_pixel" type="bool" setter="set_snap_2d_vertices_to_pixel" getter="is_snap_2d_vertices_to_pixel_enabled" default="false">
 		</member>
+		<member name="texture_mipmap_bias" type="float" setter="set_texture_mipmap_bias" getter="get_texture_mipmap_bias" default="0.0">
+			Affects the final texture sharpness by reading from a lower or higher mipmap (also called "texture LOD bias"). Negative values make mipmapped textures sharper but grainier when viewed at a distance, while positive values make mipmapped textures blurrier. Recommended values are between [code]-0.75[/code] and [code]0.0[/code]. Enabling temporal antialiasing ([member ProjectSettings.rendering/anti_aliasing/quality/use_taa]) can help reduce the graininess visible when using negative mipmap bias.
+			There is usually not much reason to use positive mipmap bias, except for troubleshooting purposes.
```

Comment:
I would remove this line. We already say that positive values make textures blurrier. It isn't helpful to users to say that there is no reason to use a given setting

Reply:
I was wondering whether the property hint should even allow using positive values in the first place. People might think it improves performance or reduces memory usage when it does neither (in most cases).

Reply:
It can improve performance in scenes that are bottle necked by heavy use of textures. Note, that this setting is different than in https://github.com/godotengine/godot/pull/38569/files as it can be changed at run time so it can be more useful for users who are looking to use it to gain a temporary performance improvement, or to achieve a particular effect/art style.